docs/quick-start.md

```diff
@@ -296,30 +296,19 @@ Selectors also need to be separated by spaces.
 Only a single matcher can be used in a message. Matchers can contain multiple
 values to match against, and can contain multiple variants.
 
-## Declarations
+## Local Declarations
 
-A declaration assigns a value to a variable within a message. This variable can then be used in other expressions within the same message. Declarations are optional: many messages will not contain any declarations.
+A **local** declaration binds a variable to the value of an expression. `.local` is like `const` in JavaScript. Think of it as a convenient way to reuse the result of an operation without repeating it.
 
-An **input** declaration binds a variable to an external input value.
-
-A **local** declaration binds a variable to the value of an expression. `.local` is like `const` in JavaScript.
-
-**EXAMPLE**
-```
-.input {$x :number style=percent}
-.local $y = {|This is an expression|}
-{{$y}}
+```mf2
+.local $x = {|This is an expression|}
+.local $y = {$now :datetime dateStyle=long}
 ```
 
-This message works without errors as long as `x` is provided as an external input value. If so, then the annotation in the `.input` declaration is applied to the value passed in for `x`: it's formatted as a number with the style "percent".
+## Input Declarations
 
-`.input` declarations are optional, so a variable that appears without a declaration is assumed to be an external input value.
+An **input** declaration binds a variable to an external input value. It's not required in order to use external input but is a declarative way to verify the presence of (and even the type of) any input.
```

Comment:
Good! Can you mention that it doesn't just check the type and presence, but can also be used to apply specific formatting to all uses of an input variable (like you do below with `style=percent`). Maybe actually two examples - one showing it just with `:number` with no options, then a paragraph explaining you can also use it to change formatting, and then an example showing how `style=percent` changes the formatting of the variable